aks/helm.tf

```diff
@@ -13,5 +13,3 @@ provider "helm" {
   namespace       = "${kubernetes_service_account.tiller.metadata.0.namespace}"
   tiller_image    = "gcr.io/kubernetes-helm/tiller:v2.14.0"
 }
-
-
```

Comment:
I don't understand why we are deleting stuff from aks.

Reply:
This change was unintentional, reverted it back.